_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+  - name: Andrew MacDonald
+    url: http://aammd.info/
+  - name: Kara Woo
+    url: http://karawoo.com/
+  - name: Carl Boettiger
+    url: http://www.carlboettiger.info/
+  - name: Jeff Hollister
+    url: https://jwhollister.com/
+date: "2017-08-01"
+output:
+  md_document:
+    variant: markdown_github
+    toc: true
+---
+
+We all know we should document our data, and some documentation is better than no documentation.  Even better, it's best practice to write metadata, i.e. structured documentation, for a dataset based on a standard machine-readable metadata format rather than in an unstructured document. A well-established standard is the [Ecological Metadata Language (EML)](https://knb.ecoinformatics.org/#external//emlparser/docs/index.html). If you follow the standard properly, you're sure to have documented your data well, which gives you a good feeling. Moreover, this will allow easier sharing of that dataset later. 
+
+That said, it's not always fun to write standard metadata. Although there's a very good package for helping you create the EML, [rOpenSci's EML package](https://github.com/ropensci/eml), documenting the data can be quite tedious.  Furthermore, before you share the data on a public repository that enforces EML, the only prize you get is a happy conscience, which isn't very tangible. In our unconf project, we created immediate gratification for EML users: a package that transforms the non-human readable EML file into a pretty documentation website for any dataset! 
+
+# What's EML, exactly, and why can't a human read it?
+
```

Comment:
"What's EML, exactly, and why can't a human read it"

suggestion: why can't humans read it?